src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1071,7 +1071,7 @@ internal static bool IsTrue(IList objectArray)
         /// </summary>
```

Comment:
@vexx32, your last commit had 4 failures in `PowerShell-CI-macos`
Null Representatives.Numeric Comparisons.{ $null } should be considered greater than or equal to 0
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Language/Scripting/NullRepresentatives.Tests.ps1: line 103
103:             $Value.InvokeReturnAsIs() -ge 0 | Should -BeTrue
```
Null Representatives.Numeric Comparisons.{ [DBNull]::Value } should be considered greater than or equal to 0
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Language/Scripting/NullRepresentatives.Tests.ps1: line 103
103:             $Value.InvokeReturnAsIs() -ge 0 | Should -BeTrue
```
Null Representatives.Numeric Comparisons.{ [NullString]::Value } should be considered greater than or equal to 0
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Language/Scripting/NullRepresentatives.Tests.ps1: line 103
103:             $Value.InvokeReturnAsIs() -ge 0 | Should -BeTrue
```
Null Representatives.Numeric Comparisons.{ [AutomationNull]::Value } should be considered greater than or equal to 0
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Language/Scripting/NullRepresentatives.Tests.ps1: line 103
103:             $Value.InvokeReturnAsIs() -ge 0 | Should -BeTrue
```